<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCStackedSettingsFrame</class>
 <widget class="QFrame" name="ICHCStackedSettingsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <widget class="QLabel" name="currentPageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="page1ToolButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>111</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Group-2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="page0ToolButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>38</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Group-1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="page2ToolButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>184</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Group-3</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="page3ToolButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>257</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Group-4</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>217</y>
     <width>106</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="yRPStepLineEdit">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>282</y>
     <width>146</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="xRPLatticeLineEdit">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>244</y>
     <width>146</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="xRPStepLineEdit">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>244</y>
     <width>146</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="yRPLatticeLineEdit">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>282</y>
     <width>146</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="zRPLatticeLineEdit">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>320</y>
     <width>146</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="zRPStepLineEdit">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>320</y>
     <width>146</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>245</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>282</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>244</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>282</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>320</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>217</y>
     <width>116</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lattice</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>15</y>
     <width>231</width>
     <height>198</height>
    </rect>
   </property>
   <property name="title">
    <string>Sequence</string>
   </property>
   <widget class="QCheckBox" name="xzyCheckBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>X-Z-Y</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="zxyCheckBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>72</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>Z-X-Y</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="yzxCheckBox">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>72</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>Y-Z-X</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="yxzCheckBox">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>25</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>Y-X-Z</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>15</y>
     <width>271</width>
     <height>198</height>
    </rect>
   </property>
   <property name="title">
    <string>Direction</string>
   </property>
   <widget class="QCheckBox" name="xPPCheckBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>X PP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="xRPCheckBox">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>25</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>X RP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="yPPCheckBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>Y PP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_4</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="yRPCheckBox">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>90</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>Y RP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_4</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="zPPCheckBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>155</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>Z PP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_5</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="zRPCheckBox">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>155</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }
</string>
    </property>
    <property name="text">
     <string>Z RP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_5</string>
    </attribute>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>72</y>
      <width>229</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>137</y>
      <width>229</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
